installing/installing_ibm_powervs/installation-config-parameters-ibm-power-vs.adoc

```diff
@@ -0,0 +1,12 @@
+:_content-type: ASSEMBLY
+[id="installation-config-parameters-ibm-power-vs"]
+= Installation configuration parameters for {ibmpowerProductName} Virtual Server
+include::_attributes/common-attributes.adoc[]
+:context: installation-config-parameters-ibm-power-vs
+:platform: {ibmpowerProductName} Virtual Server
+
+toc::[]
+
+Before you deploy an {product-title} on {ibmpowerProductName} Virtual Server, you provide parameters to customize your cluster and the platform that hosts it. When you create the `install-config.yaml` file, you provide values for the required parameters through the command line. You can then modify the `install-config.yaml` file to customize your cluster further.
```

Comment:
you provide > you must provide
//////////

you can provide values for the required parameters

Reply:
```suggestion
Before you deploy an {product-title} on {ibmpowerProductName} Virtual Server, you provide parameters to customize your cluster and the platform that hosts it. When you create the `install-config.yaml` file, you provide values for the required parameters through the command line. You can then modify the `install-config.yaml` file to further customize your cluster.
```